skelebot/systems/execution/docker.py

```diff
@@ -103,6 +124,10 @@ def push(config, host=None, port=None, user=None, tags=None):
     tags = [] if tags is None else tags
     tags = tags + [config.version, "latest"]
 
+    status = 0
     for tag in tags:
-        execute(TAG_CMD.format(src=imageName, image=image, tag=tag))
-        execute(PUSH_CMD.format(image=image, tag=tag))
+        status = execute(TAG_CMD.format(src=imageName, image=image, tag=tag))
+        status = execute(PUSH_CMD.format(image=image, tag=tag))
```

Comment:
It may be a good idea to add a check for a failed push here

Reply:
The execute function itself raises an exception if it fails. returning the status is actually kinda redundant... I would like to totally revisist error handling in v2.0.